install.sh

```diff
@@ -3,9 +3,9 @@
 # Install script for gosplat
 #
 # Updating and moving dependencies to $HOME/.local/share/
-# TODO: curl model `.bin` file from somewhere...
-mkdir -p $HOME/.local/share/gosplat
-cp -r ./python_helper $HOME/.local/share/gosplat
+./src/modelGetter/modelGetter
 
+mkdir -p $HOME/.local/share/gosplat
+rsync -av --progress ./* $HOME/.local/share/gosplat --exclude ./.git
```

Comment:
borde inte typ `cp -r src/python_helper/ $HOME/.local/share/gosplat` räcka?

Reply:
vi kan egentligen dra ned på bara the necessary stuff, som t.ex modellen o h python_helpers very true. Kan fixa det om jag har tid i en framtida uppdatering